tests/MongoDB.Driver.Tests/Specifications/crud/CrudTestRunner.cs

```diff
@@ -52,11 +53,18 @@ public class CrudTestRunner
 
         [SkippableTheory]
         [ClassData(typeof(TestCaseFactory))]
-        public void RunTestDefinition(BsonDocument definition, BsonDocument test, bool async)
+        public void Run(JsonDrivenTestCase testCase)
```

Comment:
This is a minor change to:

1. Integrate better with the Visual Studio test runner (e.g. be able to select a single test and run it)
2. Be able to use the standard TestCaseFactory